CMakeLists.txt

```diff
@@ -5,8 +5,7 @@ include("common.cmake")
 # Haywire
 # ----------------------------------------
 project(haywire C)
-set(CMAKE_BUILD_TYPE RelWithDebInfo)
-#set(CMAKE_BUILD_TYPE Debug)
+set(CMAKE_BUILD_TYPE Release)
```

Comment:
Yeah lets keep release with debug info. I talked to a bunch of people to just gauge what is common and it seems the sentiment is that if there is a crash having a stack trace is valuable.